tokio/src/syscall/mod.rs

```diff
@@ -0,0 +1,42 @@
+//! The [syscall] module is intended to provide a centralized location
+//! for interacting with OS resources such as disks and network.
+//!
```

Comment:
This should probably document that it is unstable and requires `--cfg tokio_unstable`.